IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_Maintenance/DocPropertySet.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_Maintenance" UniqueId="a5ccf5dc-9c2d-8761-9e7d-06206c294ae5" ApplicableType="IfcElement,IfcAsset,IfcSystem" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
right this property set is a mashup of many distinct concepts, assessments, maintenance tasks & installation properties. we did alot of research with a few operators in P&W about how to manage maintenance data. it comes out at three levels: 

1. low level PSets like this example
2. structure group of PSets elaborating a maintenance strategy - P&W did this one!
3. Full blown use of Work Scheduling, tasks & work orders related to elements, systems & assets.

 first point is to remember the same property cannot be applied to an occurence multiple times unless in differently named property sets.

### Maintenance properties
MaintenanceDate, NextMaintenanceDate, MaintenanceType, and possibly DisassemblyDate are already covered by the combination of `Pset_MaintenanceStrategy` & `Pset_MaintenanceTriggerDuration` which is what this was designed for.

### Installation properties
these should be grouped into there own property set 

### Assessments
PSet_Condition is the primary property set for providing the latest condition attached to an element. but as the starting caveat this is overridden every time a new assessment is made. to keep track of assessments and there details, an wider work needs to be conducted around IfcTask and Inspection information.

### Repairs 
A similar latest repair property set like PSet_Condition might be useful to cover repair properties.

Reply:
I have also looked at Pset_Condition and it should probably get a couple of these properties.

I do agree with the assessment above.